### PR TITLE
docs(Button): rename filename and class name

### PR DESCRIPTION
**Pull request by dpkwhan**

### CODE REVIEW

**Review by levithomason — APPROVED** [review]
Looks good thanks

**Comment by levithomason** [comment]
In order to keep the changes separate (Button and Container), you'll need to create separate branches with each change, or wait until the Button is merged before committing the Container changes.

I need to wait for tests before merging, but each commit will trigger tests to rerun.

**Comment by dpkwhan** [comment]
@levithomason I will create separate branch for each element. Thanks for the advice.

**Pull request closed without merging**
